packages/strapi-mongoose/lib/index.js

```diff
@@ -69,8 +71,14 @@ module.exports = function (strapi) {
 
         connectOptions.ssl = ssl === true || ssl === 'true';
 
+        options.debug = debug === true || debug === 'true';
```

Comment:
Tiny Tip: you can use `Boolean(debug)` in this case which will do exactly what you're doing here

Reply:
Thanks! updated, also changed `ssl` :)